screens/stores/StoreListScreen.js

```diff
@@ -9,7 +10,7 @@ class StoreListScreen extends React.Component {
   constructor(props) {
     super(props);
     const { stores, navigation } = this.props.navigation.state.params;
-    this.state = { stores, navigation };
+    this.state = { stores, navigation, search: '', searchStores: stores };
```

Comment:
Nit: Can we name these a little better? `stores` is probably better off as `allStores` since it's continuously being used in the `.filter`; `searchStores` --> `searchedStores`, or `filteredStores`, or anything else that captures what it is a little better